package.json

```diff
@@ -78,8 +78,10 @@
     "walletlink": "^2.1.8"
   },
   "resolutions": {
-    "ansi-regex" : "^5.0.1",
-    "axios": "^0.21.1"
+    "ansi-regex": "^5.0.1",
+    "axios": "^0.21.1",
+    "readable-stream": "2.3.7",
+    "keccak": "3.0.1"
```

Comment:
This not necessary actually. See here: https://github.com/snapshot-labs/snapshot/pull/1101